examples/portable_dashboards_example/public/static_by_reference_example.tsx

```diff
@@ -41,11 +41,13 @@ export const StaticByReferenceExample = ({
       </EuiText>
       <EuiSpacer size="m" />
       <EuiPanel
+        className="eui-scrollBar"
         hasBorder={true}
-        // By specifying the height of the EuiPanel, we make it so that the dashboard height is
+        // By specifying the height + overflow of the EuiPanel, we make it so that the dashboard height is
         // constrained to the container - so, the dashboard is rendered with a vertical scrollbar
         css={css`
           height: 600px;
+          overflow-y: auto;
```

Comment:
Because the Dashboard viewport is no longer constrained to the height of its container, we need the **container** (in this case, the `EuiPanel`) to be responsible for handling the overflow.